index.html

```diff
@@ -773,7 +773,7 @@
 			<li>If the pointer is not in the <a>active buttons state</a>, then terminate these steps.</li>
 			<li>For the specified <code>pointerId</code>, set the <dfn>pending pointer capture target override</dfn> to the <code>Element</code> on which this method was invoked.</li>
 		</ol>
-		<div class="note">When pointer capture is set, <code>pointerover</code>, <code>pointerout</code>, <code>pointerenter</code>, and <code>pointerleave</code> events are only generated when crossing the boundary of the element that has capture as other elements can no longer be targeted by the pointer. This has the effect of suppressing these events on all other elements.</div>
+		<div class="note">When pointer capture is set, <code>pointerover</code>, <code>pointerout</code>, <code>pointerenter</code>, and <code>pointerleave</code> events are only generated when crossing the boundary of the element that has capture as other elements can no longer be targeted by the pointer. This has the effect of suppressing these events on all other elements. The region of the capturing element includes the element itself and all its children to mimic the hover state of the capturing element. Moving out of (into) this region causes <code>pointerout</code> and <code>pointerleave</code> (<code>pointerover</code> and <code>pointerenter</code>) to be sent to the capturing element and all its ancestors.</div>
```

Comment:
Sure. I posted a newer version of my patch which hopefully I explained the pointerover vs pointerenter behavior better. We can discuss it again in the next meeting. Basically there are two interpretation of the capturing I can imagine regarding the transition events.

1. The whole document and all the ancestors of the capturing element shrink to the borders of the capturing element.
2. The parent element of the capturing element expands to become the whole document and you can never leave it while pointer is captured. So you can only leave the capturing element but you always stay in your parent. This second approach is I believe what you think but looking at this then I kind of expect to get the pointer over for my parent as soon as I leave the capturing element while the other interpretation doesn't have this quirk.